Makefile

```diff
@@ -19,7 +19,7 @@ GOVERSION := 1.9.3
 PROJECT := $(CURRENT_DIR:$(GOPATH)/src/%=%)
 OWNER := $(notdir $(patsubst %/,%,$(dir $(PROJECT))))
 NAME := $(notdir $(PROJECT))
-VERSION := 0.1.3
+VERSION := 0.1.4
```

Comment:
0.1.4 already exists as a release.  This supposed to be 0.1.5?

Reply:
Yeah, updated it.